src/Parser.js

```diff
@@ -682,11 +711,17 @@ Parser.prototype.parseColorGroup = function(optional) {
  * Parses a size specification, consisting of magnitude and unit.
  */
 Parser.prototype.parseSizeGroup = function(optional) {
-    var res = this.parseStringGroup("size", optional);
+    var res;
+    if (!optional && this.nextToken.text !== "{") {
+        res = this.parseRegexGroup(
+            /^[-+]? *(?:$|\d+|\d+\.\d*|\.\d*) *[a-z]{0,2}$/, "size");
```

Comment:
Also,** `.` by itself is a valid size?  (`\\kern.px`)  If so, may be interesting to add a test-case for that.